src/util.ts

```diff
@@ -221,3 +222,23 @@ export function getFavicon(): string | null {
   }
   return `${protocol}//${host}${href}`;
 }
+
+export function createQrUrl(
+  sessionId: string,
+  sessionSecret: string,
+  serverUrl: string,
+  isParentConnection: boolean
+): string {
+  const sessionIdKey = isParentConnection ? "parent-id" : "id";
+
+  const query: string = stringify({
+    [sessionIdKey]: sessionId,
+    secret: sessionSecret,
+    server: serverUrl,
+    v: "1"
```

Comment:
how about?
```suggestion
  isParentConnection: boolean,
  version = "1"
): string {
  const sessionIdKey = isParentConnection ? "parent-id" : "id";

  const query: string = stringify({
    [sessionIdKey]: sessionId,
    secret: sessionSecret,
    server: serverUrl,
    v: version
```

Reply:
you could even inline it `[isParentConnection ? 'parent-id' : 'id']: sessionId,`

Reply:
i think keeping the ternary separate will be a little more readable:)